ddl/ddl_api.go

```diff
@@ -673,6 +673,17 @@ func getDefaultValue(ctx sessionctx.Context, col *table.Column, c *ast.ColumnOpt
 
 		return value, nil
 	}
+	// handle default next value of sequence. (keep the expr string)
+	str, isSeqExpr, err := handleSequenceDefaultValue(c)
+	if isSeqExpr {
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		col.DefaultIsExpr = true
```

Comment:
It's strange to modify the column in a "get" function,  can we avoid it?

Reply:
done.